compose-builder/add-device-s7.yml

```diff
@@ -0,0 +1,42 @@
+# /*******************************************************************************
+#  * Copyright 2024 YIQISOFT.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+
+version: '3.7'
+
+services:
+  device-s7:
+    image: ${DEVICE_SVC_REPOSITORY}/device-s7${ARCH}:${DEVICE_S7_VERSION}
+    ports:
+      - '127.0.0.1:59994:59994'
```

Comment:
Was this port # officially assigned be the device services WG? @lindseysimple ??

Reply:
@lenny-intel I've updated the Device WG [wiki](https://wiki.edgexfoundry.org/display/FA/Device+and+Device+SDK+Working+Group) to assign 59994 port to S7, is there any other steps to be done? Thanks.

Reply:
@lindseysimple , the docs here also need to be updated.
https://docs.edgexfoundry.org/3.1/general/ServicePorts/

IMO, the wiki should point to this page rather than be a full duplicate so we don't have to update both of them each time.